test/internet.spec.ts

```diff
@@ -87,7 +85,10 @@ describe('internet', () => {
 
           const [prefix, suffix] = email.split('@');
 
-          expect(prefix).toMatch(/^Aiden.Harann55/);
+          expect(prefix).includes('Aiden.Harann55');
+          expect(prefix).toMatch(
+            /^(Aiden\.Harann55((\d{2})|([._][A-Za-z]*\d{2})|([._][A-Za-z]*)))/
```

Comment:
Couldn't this be shortened to the following?
```suggestion
            /^(Aiden\.Harann55((\d{2})|([._][A-Za-z]*(\d{2})?)))/
```

Reply:
sure could, I'll commit the change